scheds/rust/scx_lavd/src/main.rs

```diff
@@ -201,6 +202,12 @@ struct Opts {
     /// Show descriptions for statistics.
     #[clap(long)]
     help_stats: bool,
+
+    /// Enable virtual LLC partitioning. Use without arguments for default limits
+    /// (2-8 cores per partition), or specify min,max values like --virt-llcs=6-12
+    /// for custom partition limits.
+    #[clap(long = "virt-llcs", value_delimiter = '-', num_args = 0..=2)]
```

Comment:
I wonder whether providing similar common option mechanism like `LibbpfOpts` would be better so that individual schedulers don't have to deal with custom options.